pycaret/internal/tabular.py

```diff
@@ -3257,21 +3289,74 @@ def create_model_supervised(
             logger.info("Finalizing model")
             with io.capture_output():
                 pipeline_with_model.fit(data_X, data_y, **fit_kwargs)
+            
+            # calculating model results with final model on complete train dataset
```

Comment:
@Yard1  Could you please review this section? Can we simply get the metrics by doing a pull after the final fit or do we need to do all these steps?

Reply:
We should be able to use `create_model` here with set data arguments and then `pull`